tests/actor/_power_managing/_sorted_set.py

```diff
@@ -0,0 +1,39 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Tests for the sorted set."""
+
+from frequenz.sdk.actor._power_managing import _sorted_set
+
+
+def test_sorted_set() -> None:
```

Comment:
I guess this is a pretty good candidate for *hypothesis* (many inputs that could be tested and just a few simple properties to test for).